### ISSUE TITLE
scope.generatorType undefined

### ISSUE THREAD

**Issue by bardlehel** [balderdashy/sails-generate-backend#28]
Hey,

Not sure exactly how to use this package as I'm still a bit new to Sails.  After copying the project to a local directory, I have run: 
npm install
node ./bin

but it complains about the scope.generatorType not being set.  Where do I set this or do I need to install this generator in a different way?

**Comment by konstantinzolotarev** [comment]
@bardlehel Ohh this actually a bit complicated. We don't have tests here. So I'm testing it using `npm link` command. Just link this module into my global `sails` module. and it uses needed generator when I call `sails new someProject`

Did you use `npm link` before ?

**Comment by bardlehel** [comment]
i haven't.  can you link me the proper documentation or give me run-down on steps i'd need to perform?

**Comment by konstantinzolotarev** [comment]
@bardlehel Actually there is no exact step by step documentation for sails-generators. Here are links that will help with linking modules.
https://docs.npmjs.com/cli/link
https://60devs.com/simple-way-to-manage-local-node-module-using-npm-link.html

**Comment by sailsbot** [comment]
Thanks for posting, @bardlehel.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
